source/lib/vagrant-openstack-provider/config.rb

```diff
@@ -67,10 +67,15 @@ class Config < Vagrant.plugin('2', :config)
       # @return [String]
       attr_accessor :username
 
-      # The domain name to access Openstack, this defaults to Default.
+      # The user domain name to access Openstack, this defaults to Default.
       #
       # @return [String]
-      attr_accessor :domain_name
```

Comment:
We should keep attribute `domain_name` alongside `user_domain_name` and `project_domain_name` for backward compatibility. If not, after a plugin update, it will break Vagrantfiles for people using Keystone v3 API with `domain_name` attribute.